molr-mole-server/src/main/java/io/molr/mole/server/rest/MolrMoleRestService.java

```diff
@@ -51,22 +50,22 @@ public MolrMoleRestService(Mole mole) {
         return mole.parameterDescriptionOf(new Mission(missionName)).map(MissionParameterDescriptionDto::from);
     }
 
-    @GetMapping(path = MoleWebApi.INSTANCE_STATES_PATH, produces = MediaType.APPLICATION_STREAM_JSON_VALUE)
+    @GetMapping(path = INSTANCE_STATES_PATH, produces = MediaType.TEXT_EVENT_STREAM_VALUE)
```

Comment:
This one is also more a question to @michi42  and @andreacalia: Do we really like these constants? About 5 years ago would have said that they should be like they are (as they are constants used by the client and the server).... However now I tend more to say that they make the code more complicated to read/understand, as one does not see directly the REST API just from this file, but has to look up a file, which is even in a different project... (I know IDEs make it easy ... still).
(just to say: No strong vote here, just for thought ;-)

Reply:
That is a good question.. as always no clear answer I'm afraid hahah
I agree with your comment, maybe we can find another way to specify this without hardcoding here the URL. The idea is that since we also provide the client for molr in Java we would like to make sure the url are the same (avoiding refactoring mistakes).
Probably it could be better to use some kind of configuration file/property files/whatever in which we can list all the REST endpoints that we would like to expose.. being this a text file it can be used for other language implementations then :)
Do you think we could completely get rid of the central place for storing the endpoints urls? I'm just afraid that at some point we could silently break the endpoints without noticing.. (url format for example).
Another option could be to autogenerate this file based on the spring configuration.. this could help a bit since this is guaranteed to be present in the REST api (in the other case one could forget to update the web api file for example..)
@michi42 what do you think? :smile: